frontend/packages/kubevirt-plugin/src/utils/validations/vm/nic.ts

```diff
@@ -1,6 +1,8 @@
 import { EntityMap } from '@console/shared';
-import { validateDNS1123SubdomainValue, ValidationErrorType } from '../common';
-import { addMissingSubject } from '../../grammar';
+import { isValidMAC } from 'kubevirt-web-ui-components';
```

Comment:
can you please port the validation function here and remove the `kubevirt-web-ui-components` dependency?

Reply:
I can, but that would mean maintaining it in both locations. We already depend on `kubevirt-web-components` in the plugin, so I'm unclear as to what's gained by duplicating the functionality here? As long as there's an appropriate advantage to doing it then I'm happy to do so.

Reply:
We are deprecating `kubevirt-web-ui-components`. It is an ongoing effort to move the whole code base here.

Reply:
Ah, ok. I misunderstood, then. I thought we were still going to maintain parts of the code base there. You're 100% correct, then. I'll get the validation function ported over, as well. =)

Reply:
Done!